integration_test/pcb/integration_test_board/eagle.epf

```diff
@@ -0,0 +1,25 @@
+[Eagle]
+Version="06 03 00"
+Platform="Windows"
+Serial="62191E841E-LSR-WLM-1EL"
+Globals="Globals"
+Desktop="Desktop"
+
+[Globals]
+AutoSaveProject=1
+UsedLibrary="C:/Users/richa/Documents/eagle/lbr/Richard/diode.lbr"
```

Comment:
You might want to include the package libraries here as well, if they are external

Reply:
Those are my private libraries. As far as I know, they are not needed to view and modify the schematics.